crates/wasmtime/src/runtime/component/component.rs

```diff
@@ -834,9 +834,19 @@ impl Component {
     pub(crate) fn export_lifted_function(
         &self,
         export: ExportIndex,
-    ) -> (TypeFuncIndex, &CoreDef, &CanonicalOptions) {
+    ) -> (
+        TypeFuncIndex,
+        &CoreDef,
+        ModuleInternedTypeIndex,
+        &CanonicalOptions,
+    ) {
         match &self.env_component().export_items[export] {
-            Export::LiftedFunction { ty, func, options } => (*ty, func, options),
+            Export::LiftedFunction {
+                ty,
+                func,
+                func_ty,
```

Comment:
Given that this live in `CanonicalOptions` I don't think there's any need for it to live in `LiftedFunction` any more?